frontend/workflows/k8s/src/resize-hpa.tsx

```diff
@@ -74,6 +74,10 @@ const HPADetails: React.FC<WizardChild> = () => {
           {
             name: "Min Size",
             value: hpa.sizing.minReplicas,
+            textFieldLabels: {
+              disabledField: "Current size",
```

Comment:
I think Current Min, New Min is easier to follow so each row has different labels

Reply:
updated

<img width="500" alt="Screen Shot 2022-10-25 at 1 55 08 PM" src="https://user-images.githubusercontent.com/39421794/197847152-85cf69af-1a14-4f1f-90fc-9e78162f032d.png">